.github/workflows/macos_ci.yml

```diff
@@ -30,11 +30,6 @@ jobs:
         with:
           path: ~/.cargo/git
           key: ${{ runner.os }}-cargo-index-${{ hashFiles('**/Cargo.lock') }}
-      - name: Cache cargo build
-        uses: actions/cache@v1
```

Comment:
Does the `v2` release fix it?

https://github.com/actions/cache

Reply:
2.1.4 has the fix, but it seems there's some sort of gtar issue and i believe the v2 tag doesn't point at it -- https://github.com/actions/cache/issues/528 . A few rust repos from the issues look like they have manually pinned to 2.1.4 to get the original issue fixed. So we could probably try that here alternatively, assuming the gtar issue doesn't happen?

Reply:
I see. Ok, I'll just merge this for now then